### PR TITLE
Refactor "cancel" to reduce cognitive complexity

### PR DESCRIPTION
**Pull request by lucasrmendonca**
### Description, Motivation and Context
Refactor "cancel" to reduce cognitive complexity

### What kind of change does this PR introduce?

- [x] Refactoring

### Checklist:
- [x] My code follows the code style of this project.
- [ ] I have added tests to cover my changes.
- [x] Overall test coverage is not decreased.
- [x] All new and existing tests passed.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/wampy.js [hunk 1]
```diff
@@ -2061,51 +2061,44 @@ class Wampy {
...
      *                                  "skip" | "kill" | "killnowait". Skip is default.
      *                          }
      *
      * @returns {Boolean}
      */
-    cancel (reqId, advancedOptions) {
-        const options = {};
-
-        if (!this._preReqChecks(null, 'dealer')) {
+    cancel (requestId, advancedOptions) {
```
Comment (KSDaemon) [inline src/wampy.js:2072]: Hmm... Well, I don't mind about this renaming. But you changed it only in one function. But there are still 60+ `reqId`s that exists across sources. Also `reqId` is a part of the public status object. I'm for consistency.

**Review by KSDaemon — COMMENTED** [review]
And another one question left

File: src/wampy.js (hunk 1 above)
Comment (lucasrmendonca) [reply src/wampy.js:2072]: Ok I will change it back then

**Comment by KSDaemon** [comment]
LGTM! Merging!

**Pull request merged**